roottest/root/graf/testImageFormats.cxx

```diff
@@ -0,0 +1,22 @@
+#include <TGClient.h>
+#include <TGPicture.h>
+
+#include <gtest/gtest.h>
+
+// Import a few icons from ROOT's icon directory
+
+TEST(ImageFormats, GIF)
+{
+   auto icon = gClient->GetPicture("GoBack.gif");
```

Comment:
```suggestion
   ASSERT_NE(gClient, nullptr);
   auto icon = gClient->GetPicture("GoBack.gif");
```

Reply:
`ASSERT` would still fail the test if the graphics system is not initialised. I think I have an idea, pushing now.

Reply:
(Yep, it would just give a more informative error rather than a sigsegv)